src/applications/hca/config/form.js

```diff
@@ -642,7 +654,8 @@ const formConfig = {
           title: 'Financial disclosure',
           uiSchema: {
             'ui:title': 'Financial disclosure',
-            'ui:description': financialDisclosureText,
+            'ui:description': PrefillMessage,
```

Comment:
I don't think there's any prefilled information on this page, it's just the yes/no field and a warning.

Reply:
Prefill message removed.